support/supportedversion/version.go

```diff
@@ -76,3 +80,43 @@ func IsValidReleaseVersion(version, currentVersion, latestVersionSupported, minS
 
 	return nil
 }
+
+type ocpVersion struct {
+	Name        string `json:"name"`
+	PullSpec    string `json:"pullSpec"`
+	DownloadURL string `json:"downloadURL"`
+}
+
```

Comment:
would you mind document this func? e.g `// LookupLatestSupportedRelease picks the latest multi-arch image supported by this Hypershift Operator...`